integration/resource_aws_external_id_integration.go

```diff
@@ -11,9 +11,12 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-func dataSourceExternalId() *schema.Resource {
```

Comment:
This is incorrect, it was correct previously. External id should be data source and not resource